### ISSUE TITLE
Indian phone numbers only

### ISSUE THREAD

**Issue by MohammedElattar** [venu123143/whatsapp#30]
I can't add my own number from different country

**Comment by venu123143** [comment]
yes you can. this is not paid project. this is personal one. so i have not used any 3rd party for the otp. you will get otp in the toast itself.
